qiskit/visualization/dag_visualization.py

```diff
@@ -192,12 +195,20 @@ def node_attr_func(node):
                     n["style"] = "filled"
                     n["fillcolor"] = "lightblue"
                 if isinstance(node, DAGInNode):
-                    n["label"] = bit_labels[node.wire]
+                    if isinstance(node.wire, Qubit):
+                        label = register_bit_labels.get(node.wire, f"q[{qubit_indices[node.wire]}]")
```

Comment:
Bikeshedding: is there some alternative label format we can use for registerless qubits?  In theory, if I make a circuit:
```python
qreg = QuantumRegister(2, 'q')
bits = [Qubit(), Qubit()]
qc = QuantumCircuit(bits, qreg)
dag = circuit_to_dag(qc)
```
then the visualisation will have two qubits labelled `q[0]` and `q[1]`.  Perhaps `qubit 0` (not using `[]`) to distinguish the formats?

I'm not particularly bothered by it, though.

Reply:
Yeah I thought about that too and similarly wasn't bothered by the potential conflict. I'm open to using a different label though to avoid potential confusion. Maybe `q_0` instead of `q[0]`?

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/7923/commits/c8b7ecca74420f5c213cb31a0df9b306ecadb3f9